bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/firmware/ProgressCallback.java

```diff
@@ -74,4 +75,15 @@
      * Callback operation to indicate that the firmware update was canceled.
      */
     void canceled();
+
+    /**
+     * Callback operation to update the percentage progress of the firmware update.
+     * This method can be used to provide detailed progress information additional to the sequence or even without a
```

Comment:
I would like to have a description how to use the callback interface on the class / interface level... so that binding developers see how to use it (defining a sequence vs. progress based updateds)